src/Cynthia.Card/src/Cynthia.Card.Server/Locales/en.json

```diff
@@ -408,7 +408,7 @@
         },
         "12035": {
             "Name": "Renew",
-            "Info": "Resurrect a Gold unit.",
+            "Info": "Resurrect a non-Leader Gold unit.",
```

Comment:
I think technically Leader is not counted as Gold, but maybe it is good to make it clear here?

Reply:
At least it was stated as such in the English version, but yeah, Leaders are kinda their own thing, which at least was one of the things they added for Homecoming (Cyan Border). Not an important change though :)